### PR TITLE
Now there is hover effect for sidebar #1

### PR DESCRIPTION
**Pull request by AK4266**
The issue is fixed, as Iam getting color when I hover my mouse pointer
chrome_T3btACfH5l

### CODE REVIEW

**Comment by LakshaySK106** [comment]
can you also fix,
1. the active state of the buttons in the sidebar?
2. on hover the button changed to the text (name of the page in uppercase)

**Comment by AK4266** [comment]
@LakshaySK106 Yes, I will work on those issues as well. Should I create new issue mentioning those two. So that you can assign me

**Comment by LakshaySK106** [comment]
you may come up with additional source code in this current PR or can create a new issue, your pick.

**Comment by AK4266** [comment]
@LakshaySK106  I will create new issue. Assign it to me

**Comment by AK4266** [comment]
Just raised the issue. Can you assign it to me

File: client/src/components/Sidebar.jsx [hunk 1]
```diff
@@ -4,15 +4,44 @@ import { logo, sun, profile, money, payment, homeIcon} from '../assets';
 import { navlinks } from '../constants';
 import { useAuth0 } from "@auth0/auth0-react"
 
+// const Icon = ({ styles, name, imgUrl, isActive, disabled, handleClick }) => (
```
[the remark this answers is not in the document]
Comment (AK4266) [reply client/src/components/Sidebar.jsx:7]: Nah, no particular reason, I just coded from beginning thinking that if I miss up I will have the original code. So I commented it out while coding. Should I remove it?

[the remark this answers is not in the document]
Comment (singh1203) [reply client/src/components/Sidebar.jsx:7]: It would be great if you remove it, helping to maintain clean code.

**Comment by AK4266** [comment]
Sure, I will do it right away

**Comment by LakshaySK106** [comment]
Make sure to work on the latest commit

**Comment by AK4266** [comment]
Yeah will do

**Comment by AK4266** [comment]
Hey@LakshaySK106 I just deleted the commented code from earlier, Can you take a look

**Review by singh1203 — APPROVED** [review]
LGTM.

**Comment by AK4266** [comment]
Hey @LakshaySK106 , still working on it. Need more time

**Comment by LakshaySK106** [comment]
Yeah, no worries.

**Comment by LakshaySK106** [comment]
you can create a new PR when you are done.

**Comment by AK4266** [comment]
Yeah, Thanks

**Pull request merged**